x-pack/plugins/security_solution/public/timelines/components/timeline/selectable_timeline/index.tsx

```diff
@@ -51,6 +48,16 @@ export const ORIGINAL_PAGE_SIZE = 50;
 const POPOVER_HEIGHT = 260;
 const TIMELINE_ITEM_HEIGHT = 50;
 
+/**
+ * Modifies options by creating new property `timelineTitle`(with value of `title`), and by setting `title` to undefined.
+ * Thus prevents appearing default browser tooltip on option hover (attribute `title` that gets rendered on li element)
+ *
+ * @param {EuiSelectableOption[]} options
+ * @returns {EuiSelectableOption[]} modified options
+ */
+const replaceTitleInOptions = (options: EuiSelectableOption[]): EuiSelectableOption[] =>
+  options.map(({ title, ...props }) => ({ ...props, title: undefined, timelineTitle: title }));
```

Comment:
nit: not a blocker, but the workaround to prevent the default browser tooltip doesn't work in Safari `15.5`

![safari](https://user-images.githubusercontent.com/4459398/176739687-87183806-30f2-469a-b3e6-b8072d40155b.png)

Reply:
This seems to be a default behaviour of Safari when text is trimmed. It will display native browser title description